tests/testthat/test-test_dir.R

```diff
@@ -17,13 +25,17 @@ test_that("test_dir()", {
   df <- as.data.frame(res)
   df$user <- df$system <- df$real <- df$result <- NULL
 
+  # We order this here, because the order of parallel tests is
+  # not fully predictable.
+  df <- df[order(df$file, df$context,  df$test), ]
```

Comment:
Don't you really only need to order by file? The order within a file is still deterministic.

Reply:
Yeah, but `order()` might not be stable. The man page suggests that it is, but links to `sort()` and there they say it is only stable if the vector has names. So it is better to be on the safe side.

Reply:
I think you should just assume it's stable; otherwise sorting by context and test are going to cause a lot of ordering changes. (If it later turns out not to be the case, we can add an integer index to tests to track the exact order)

Reply:
Yes, for the actual results, I agree. But this is a test case, so I just want to make sure that we always get the same results, on all platforms. Might be overly cautious, but it also does not hurt too much...

Reply:
But of course I also don't mind, I can change it of course.